bin/utils/chain-spec-builder/Cargo.toml

```diff
@@ -15,10 +15,11 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 ansi_term = "0.12.1"
+clap = { version = "3.0", features = ["derive"] }
+rand = "0.8"
```

Comment:
This seems to be a repeating pattern, does clap 3.0 require rand 0.8? If so, I'd rather upgrade rand to 0.8.x in another PR, and then you merge master on this PR to pick up the changes.

Reply:
I don't see any problem in this.

Reply:
Technically, this is a breaking change, since the most significant version number is bumped. There may be some behavioral changes to existing functions. I haven't looked at the full changelog yet, but to be on the safe side, I would put unrelated version bumps to another PR.

Reply:
I'll revert it if you mind